README: add content about the org (#2)
perl-ide/.github: profile/README.md

<p align="center">
    <img alt="Logo" src="https://github.com/perl-ide/general/assets/453611/fabf497a-c7b9-488f-910e-5c6d1ce912a6" height="240" />
</p>

# Perl Development Environments (a.k.a. Perl IDE)

This organization has the sole focus of creating a better IDE experience for Perl users.
This **is not** an effort for creating a new Perl IDE tool and we **do not** focus on a single available IDE or
editor, like VSCode or Emacs or VIM, but we care about **any** development environment.

Perl-IDE organization is made by the Perl community looking at Perl's support quality in different development

> environments. Allowing users (individuals and companies) to rest asured they won't have issues on finding well

Reviewer: ```suggestion
environments. Allowing users (individuals and companies) to rest assured they won't have issues on finding well
```

documented and supported Perl support in any IDE/editor they choose.

> ## Origanization Goals

Reviewer: ```suggestion
## Organization Goals
```

The three following goals are the guides to our existence:

1. Bundle as many projects as possible, related to developer experience on programming in Perl, under this
   organization umbrella;
2. Track issues in different projects, not directly related to Perl (vscode, vim, nvim, textmate, language servers,
   ...), but affect Perl programmers' life quality, making sure Perl support in different tools is up to the same

>    level of quality than other languages support;

Reviewer: ```suggestion
   level of quality as seen in other languages;
```

3. Share knowledge, toolset, libraries, guides or any other material to allow overall better developer experience.

## Contributing

We accept any ideas, suggestions, questions and answers from the community, meaning that this organization is the main
source or, even, the house of many future projects.

### Discussions

If you want to start a discussion around an idea or suggestion or anything else, you're more than welcome to start

> with a [discussion](https://github.com/orgs/perl-ide/discussions). However, always make sure to presenting any

Reviewer: ```suggestion
with a [discussion](https://github.com/orgs/perl-ide/discussions). However, always make sure to present any
```

information or sources that may help others to understand your idea/suggestion/criticism better. We have enabled
discussions in this organization to allow asynchronous conversation between the members, so be patient.

### Projects

> On the other hand, in case you maintains a project that seem to meet the organization goals, don't hesitate on

Reviewer: ```suggestion
On the other hand, in case you maintain a project that seems to meet the organization goals, don't hesitate to
```

> creating a [discussion](https://github.com/orgs/perl-ide/discussions) sharing a bit of its history, goal, core

Reviewer: ```suggestion
create a [discussion](https://github.com/orgs/perl-ide/discussions) sharing a bit of its history, goal, core
```

> contributors you would like to invite and so on. Then, based on the members and other maintainers opinion the project

Reviewer: ```suggestion
contributors you would like to invite and so on. Then, based on the members' and other maintainers' opinion, the project
```

> will be able to be transferred directly from your namespace to Perl-IDE namespace.

Reviewer: ```suggestion
will be able to be transferred directly from your namespace to the Perl-IDE namespace.
```
